ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -201,22 +240,24 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
   // b) email address/Domain are not valid
   // c) Required fields are not empty
   disableSave(errors) {
-    return this.validateRequiredFields() || errors || this.fieldsNotEdited()
+    return this.hasInvalidFields() || errors || this.fieldsEdited()
```

Comment:
Based only on the new naming here, we would: disable save if fields are edited. That seems incorrect